tests/Benchmarks/System.Buffers.Experimental/RangeEnumeration.cs

```diff
@@ -1,48 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using BenchmarkDotNet.Attributes;
-
-namespace System.Buffers.Experimental.Benchmarks
-{
-    public class RangeEnumeration
```

Comment:
Should we migrate these perf tests to dotnet/performance?

cc @adamsitnik

Reply:
@ahsonkhan yes, thanks for pointing this! I've created https://github.com/dotnet/performance/issues/1002 to track it